opensuse_buildservice/debian/changelog

```diff
@@ -0,0 +1,35 @@
+notqmail (1.08-1) unstable; urgency=medium
```

Comment:
Move the "debian" directory up one level, that is where dpkg expects it and can automatically run it from a source tarball.